indico/modules/events/sessions/client/js/SessionForm.tsx

```diff
@@ -98,9 +98,9 @@ function SessionForm({
       id="session-form"
       header={header}
       onSubmit={onSubmit}
-      onClose={onClose}
       initialValues={initialValues}
       size="small"
+      {...(onClose ? {onClose} : {})}
```

Comment:
What was the reason for this? `FinalModalForm`'s `onClose` is not optional, and AFAICT nothing in this PR uses `SessionForm` without providing `onClose` anyway...